test/tests_for_sdkmr/ee2_SDKMethodRunner_test.py

```diff
@@ -710,6 +724,20 @@ def test_finish_job_with_error_message(self, condor):
         self.mongo_util.get_job(job_id=job_id).delete()
         self.assertEqual(ori_job_count, Job.objects.count())
 
+        expected_calls = {
+            "user_id": "wsadmin",
+            "app_module_name": "MEGAHIT",
+            "app_id": "run_megahit",
+            "func_module_name": "MEGAHIT",
+            "is_error": 1,
+        }
+
+        for key in expected_calls:
+            assert (
+                runner.catalog.log_exec_stats.call_args[0][0][key]
```

Comment:
Why is it necessary to set up a mock for `log_exec_stats` in the prior test but not in this one?

Reply:
which line #?